ipatests/prci_definitions/nightly_latest.yaml

```diff
@@ -1408,3 +1408,16 @@ jobs:
         template: *ci-master-latest
         timeout: 3600
         topology: *master_1repl
+
+  fedora-latest/test_backup_and_restore_TestBackupAndRestoreTrust:
+    requires: [testing-fedora/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{testing-fedora/build_url}'
+        update_packages: True
```

Comment:
remove `update_packages: True` for the nightly_latest. This tag forces dnf update of all pkgs and we enforce this only on the nightly_lastest_testing.yaml file.